rx-java/rxgrpc-stub/src/main/java/com/salesforce/rxgrpc/stub/ClientCalls.java

```diff
@@ -82,14 +96,19 @@ public void accept(Throwable t) {
      */
     public static <TRequest, TResponse> Flowable<TResponse> oneToMany(
             final Single<TRequest> rxRequest,
-            final BiConsumer<TRequest, StreamObserver<TResponse>> delegate) {
+            final BiConsumer<TRequest, StreamObserver<TResponse>> delegate,
+            final CallOptions options) {
         try {
+
+            final int prefetch = options == null ? CALL_OPTIONS_PREFETCH.getDefault() : options.getOption(CALL_OPTIONS_PREFETCH);
+            final int lowTide = getLowTide(options, prefetch);
+
             return rxRequest
                     .flatMapPublisher(new io.reactivex.functions.Function<TRequest, Publisher<? extends TResponse>>() {
                         @Override
                         public Publisher<? extends TResponse> apply(TRequest request) {
                             final RxClientStreamObserverAndPublisher<TResponse> consumerStreamObserver =
-                                new RxClientStreamObserverAndPublisher<TResponse>(null);
+                                new RxClientStreamObserverAndPublisher<TResponse>(null, null, prefetch, lowTide);
```

Comment:
Does pulling prefetch and lowTide up into client and server calls render any of the abstract base class constructors unused? If so, can you delete them?

Reply:
I can do cleanup. But not sure if there can be something external using these. What do you think?

Reply:
I believe most of the affected classes and constructors are package private.